src/app/handlers/auto_unverify.handler.ts

```diff
@@ -0,0 +1,19 @@
+import { IContainer, IHandler, IMessage } from '../../common/types';
+
+export class AutoUnverifyHandler implements IHandler {
+  constructor(public container: IContainer) {}
+
+  public async execute(message: IMessage): Promise<void> {
+    const hasAvatar = Boolean(message.author.avatar);
+    if (hasAvatar) {
+      return;
+    }
+
+    const unverifiedRole = message.guild.roles.filter((role) => role.name === 'Un verified');
+    const targetGuildMember = message.guild.members
```

Comment:
Could be slow

Reply:
Are you referring to #213 ?
I literally have like 40 roles, all called 'new role' in my test server, and the addRole is almost instant to the user joining

Reply:
https://github.com/joey-colon/lion/blob/fb013501794232b15131aae7a5c683aa42c892e1/src/app/listener.ts#L54 Try this flow to look up member